selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/PullRequestPluginWithForkTest.java

```diff
@@ -68,117 +64,97 @@
+  @Inject private ImportProjectFromLocation importWidget;
   @Inject private TestGitHubServiceClient gitHubClientService;
+  @Inject private TestProjectServiceClient testProjectServiceClient;
+  @Inject private TestUserPreferencesServiceClient testUserPreferencesServiceClient;
 
   @BeforeClass
   public void setUp() throws Exception {
-    ide.open(ws);
+    Path entryPath =
+        Paths.get(getClass().getResource("/projects/default-spring-project").getPath());
+    testAuxiliaryRepo.addContent(entryPath);
+
+    ide.open(testWorkspace);
 
     // add committer info
-    testUserPreferencesServiceClient.addGitCommitter(gitHubUsername, productUser.getEmail());
+    testUserPreferencesServiceClient.addGitCommitter(gitHubUsername, testUser.getEmail());
 
     // authorize application on GitHub
-    menu.runCommand(
-        TestMenuCommandsConstants.Profile.PROFILE_MENU,
-        TestMenuCommandsConstants.Profile.PREFERENCES);
+    menu.runCommand(PROFILE_MENU, PREFERENCES);
     preferences.waitPreferencesForm();
     preferences.generateAndUploadSshKeyOnGithub(gitHubUsername, gitHubPassword);
   }
 
-  @AfterClass
-  public void tearDown() throws Exception {
-    try {
-      gitHubClientService.deleteRepo(FORK_NAME_REPO, gitHubUsername, gitHubPassword);
-    } catch (NotFoundException e) {
-      // ignore absent repo to delete
-      LOG.debug("Repo {} is not found.", FORK_NAME_REPO);
-      return;
-    }
-
-    List<String> listPullRequest =
-        gitHubClientService.getNumbersOfOpenedPullRequests(
-            NAME_REPO, githubUserCloneName, githubUserClonePassword);
-
-    if (!listPullRequest.isEmpty()) {
-      gitHubClientService.closePullRequest(
-          NAME_REPO,
-          Collections.max(listPullRequest),
-          githubUserCloneName,
-          githubUserClonePassword);
-    }
-  }
-
   @Test
-  public void createPullRequest() {
-    explorer.waitProjectExplorer();
-    menu.runCommand(
-        TestMenuCommandsConstants.Workspace.WORKSPACE,
-        TestMenuCommandsConstants.Workspace.IMPORT_PROJECT);
-    importWidget.waitMainForm();
-    importWidget.selectGitSourceItem();
-    loader.waitOnClosed();
-    importWidget.typeURi(PROJECT_URL);
-    importWidget.typeProjectName(PROJECT_NAME);
-    importWidget.clickImportBtnWithoutWait();
-    wizard.selectTypeProject(Wizard.TypeProject.BLANK);
-    loader.waitOnClosed();
-    wizard.clickSaveButton();
-    loader.waitOnClosed();
-    wizard.waitCreateProjectWizardFormIsClosed();
-    explorer.waitItem(PROJECT_NAME);
-    explorer.openItemByPath(PROJECT_NAME);
-    explorer.openItemByPath(PROJECT_NAME + "/README.md");
+  public void createPullRequest() throws Exception {
+    String projectUrl = testAuxiliaryRepo.getHtmlUrl() + ".git";
+
+    // import project
+    projectExplorer.waitProjectExplorer();
+    menu.runCommand(WORKSPACE, IMPORT_PROJECT);
+    importWidget.waitAndTypeImporterAsGitInfo(projectUrl, PROJECT_NAME);
```

Comment:
I think, should replace the `importWidget.waitAndTypeImporterAsGitInfo()' to the `git.importJavaApp`